cert_setup/ssl-update.sh

```diff
@@ -0,0 +1,137 @@
+# This script is originally from https://gist.github.com/maxivak/4706c87698d14e9de0918b6ea2a41015
+# with some adaptations for first-run.
+
+#!/bin/bash
+
+# Edit these two values before running this script or init-certs.sh for the
+# first time.
+DOMAINS="domain.com"
+EMAIL="email@domain.com"
```

Comment:
These are valid domains, could you use e.g., example.com so that LetsEncrypt is not asked by the scripts to set up a cert for the valid default domain?

Separately, I wonder whether we could move this into an `.env`  file so it is picked up by docker-compose in a less magic way. That would also allow us to use a separate env for dev and prod so we could e.g., disable SSL for dev.